jac/jaclang/compiler/passes/main/pyast_gen_pass.py

```diff
@@ -772,7 +776,7 @@ def exit_ability(self, node: uni.Ability) -> None:
                     ),
                     body=[cast(ast3.stmt, i) for i in body],
                     decorator_list=[cast(ast3.expr, i) for i in decorator_list],
-                    returns=self.sync(ast3.Constant(value=None)),
+                    returns=ast_returns,
```

Comment:
This was a *BUG* in the py ast code gen, it currently always generate code as `-> None` this will fix, I should make another PR for this.